BitwardenShared/UI/Auth/Landing/LandingProcessor.swift

```diff
@@ -47,10 +47,12 @@ class LandingProcessor: StateProcessor<LandingState, LandingAction, LandingEffec
         self.coordinator = coordinator
         self.services = services
 
-        let rememberedEmail = services.appSettingsStore.rememberedEmail
         var state = state
-        state.email = rememberedEmail ?? ""
-        state.isRememberMeOn = rememberedEmail != nil
+        if state.email.isEmpty {
```

Comment:
just curious, is there a specific reason this condition was needed?

Reply:
If you come from the `landingSoftLoggedOut` route that I added, the email will already be set in the state which should take precedence over the remembered email. So if the email is already set/non-empty, we want to keep that rather than fetching the remembered email. This ensures that we populate the email for the logged out account.

https://github.com/bitwarden/ios/blob/84ce42ac15b5a1bb095464435cf03934ad9c5ee1/BitwardenShared/UI/Auth/AuthCoordinator.swift#L446-L455